pyproject.toml

```diff
@@ -0,0 +1,89 @@
+[tool.poetry]
+name = "orchestration"
+version = "0.1.0dev"
+description = ""
+authors = ["Szymon Szyszkowski <ss60@mib117351s.internal.sanger.ac.uk>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10, <3.11"
+apache-airflow = {extras = ["apache-beam", "async", "celery", "google", "kubernetes", "openlineage", "postgress", "sftp"], version = "^2.9.2"}
+apache-airflow-providers-google = "^10.19.0"
+loguru = "^0.7.2"
+psycopg2-binary = "^2.9.9"
```

Comment:
```suggestion
```

DEP002 'psycopg2-binary' defined as a dependency but not used in the codebase

Reply:
this one was required by the airflow image, I will move it directly there